x-pack/legacy/plugins/index_management/public/services/use_request.ts

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
I'll delete this file and adopt the shared plugin once https://github.com/elastic/kibana/pull/40777 is merged.